GeeksCoreLibrary/Components/Configurator/Services/CachedConfiguratorsService.cs

```diff
@@ -114,5 +123,11 @@ public async Task<(decimal purchasePrice, decimal customerPrice, decimal fromPri
         {
             return await configuratorsService.CalculatePriceAsync(input);
         }
+
+        /// <inheritdoc />
+        public async Task<(decimal purchasePrice, decimal customerPrice, decimal fromPrice)> CalculatePriceAsync(VueConfigurationsModel input)
+        {
+            return await configuratorsService.CalculatePriceAsync(input);
```

Comment:
Just thinking out loud here, but maybe it is actually better to cache the price? Like we could serialize the `VueConfigurationsModel` to JSON, create a hash from that and use that as the cache key. This way when multiple people are checking the same configuration, only one of them has to actually calculate the price, others can get it from cache.

Just something to think about for the future, you don't have to make that right away (or at all).